src/particle/corsika/gen_conversion_table.py

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) 2018-2022, Eduardo Rodrigues and Henry Schreiner.
+#
+# Distributed under the 3-clause BSD license, see accompanying file LICENSE
+# or https://github.com/scikit-hep/particle for details.
+"""
+Function to generate .csv conversion table from corsica to pdg_id and vice-versa.
+This script should be kept, so the table won't need to be hand-edited in the future.
+"""
+from __future__ import annotations
+
+import csv
+import pathlib
+
+from ..particle import Particle, ParticleNotFound
+from ..shared_literals import common_particles
+
+corsica_pdg_id = [
+    (1, common_particles["gamma"]),
```

Comment:
I was wondering why you used this unusual way of getting an ID, as `common_particles["omega_782"]`. OK, it certainly is easy to read :-).

Reply:
I compiled this list by hand, and this was the easiest way of translating it (relatively) fast. (particles.literal doesn't work here...)

Reply:
Fair enough, thanks.